tests/test_helper/bats/cli_ovsovn.bats

```diff
@@ -284,4 +287,22 @@ test_paths(){
     done;
 }
 
+test_disable_microovn(){
+    for container in $TEST_CONTAINERS; do
+        run lxc_exec $container "snap disable microovn"
+        assert_success
+        run lxc_exec $container "snap enable microovn"
+        assert_success
+        sleep 5
```

Comment:
We should avoid sleeps in tests whenever possible, could this perhaps be replaced by the `microovn waitready` command?